documentation/docs/connectors/coinbase.md

```diff
@@ -69,3 +71,9 @@ Read through their article below related to trading fees and discounts.
 * [What are the fees on Coinbase Pro?](https://help.coinbase.com/en/pro/trading-and-funding/trading-rules-and-fees/fees.html)
 
 Users can override the default fees by editing [`conf_fee_overrides.yml`](/advanced/fee-overrides/).
+
+### Trading Pair Limitations
+
+Coinbase Pro has trading pair limitations on certain regions.
```

Comment:
Coinbase Pro has trading pair limitations on certain regions. Some countries have access to crypto/fiat trading pairs while other countries can only access crypto/crypto trading pairs.

Running Hummingbot with this connector on a pair that your country has no access to will result to this error:

```
OSError: Error fetching data from https://api.pro.coinbase.com/orders.
HTTP status is 400. {'message': 'Trading pair not available'}
```